tests/test_petsc.py

```diff
@@ -20,6 +20,7 @@
 
 
 @skipif('petsc')
+@pytest.mark.order(0)
```

Comment:
Adding a dependency for only this seems a bit overkill, can't this be made a pytest fixture instead so it's initialized before the tests start?